gluetool/action.py

```diff
@@ -81,11 +81,11 @@
 
 DEFAULT_TRACING_SERVICE_NAME = 'gluetool'
 DEFAULT_TRACING_REPORTING_HOST = '127.0.0.1'
-DEFAULT_TRACING_REPORTING_PORT = 5775
-DEFAULT_TRACING_FLUSH_TIMEOUT = 30
+DEFAULT_TRACING_REPORTING_PORT = '5775'
+DEFAULT_TRACING_FLUSH_TIMEOUT = '30'
```

Comment:
These values represent port *number* and timeout, which is in seconds - number again. Why changing them to strings?

Reply:
@happz yes, there was a pylint error:

https://github.com/PyCQA/pylint/issues/1669

Reply:
Then pylint is wrong. I'm not masking numbers as strings to make some linter happy. It's a number, and Python luckily does have a data type for handling numbers.

Reply:
ack

Reply:
@happz added ignore for the warning